config/routes.rb

```diff
@@ -2,6 +2,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   namespace :api do
     namespace :v1 do
+      resources :garden_healths
       resources :gardens
       resources :users
     end
```

Comment:
This pr looks good! I just have a question. I nested my resources in my pr and I'm wondering if that's unnecessary? Like wouldn't we want to route to a specific user's gardens/plants/sensors? or is current user responsible for that?

Reply:
I think these are just basic ReSTful endpoints. As long as we serialize the relationships when we call onto Gardens we should be able to get that gardens sensors. Then we can use the IDs from there to hit up a specific sensor if we need to.

Reply:
Okay, I'll change my routes up to match this. Then we can change them in the future if need be.

Reply:
Logan, I had the same thought and decided to go this way since it matched what we did on rails_engine. Eventually, we may want to add relationship endpoints and search endpoints.